product.php

```diff
@@ -0,0 +1,185 @@
+<?php include 'includes/header.php'; ?>
+
+
+<!--import css skeleton-->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css"/>
+
+<main>
+<!--Internal Product Navigation Links-->
+    <div class="row"> <!--one-->
+        <div class="twelve columns">
+            <nav>
+                <a href="">Home</a> >>
+                <a href="">Dogs</a> >>
+                <a href="">Food</a> >>
+                <a href="">Desserts</a>
+            </nav>
+        </div>
+    </div>
+
+    <!--Product Name-->
+    <div class="row"> <!--two-->
+        <div class="six columns">
```

Comment:
If this is the only element in this row you could add `one column` instead of `six columns`. We may have longer product names and they could potentially get cramped in six columns